code/index.html

```diff
@@ -8,18 +8,18 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
-    <title>Chat Bot</title>
+    <title>Welcome to Movie-Bob</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <h1>Welcome to Movie-Bob.</h1>
+    <h2>I help you select a movie for tonight.</h2>
     <main>
       <section class="chat" id="chat"></section>
 
       <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <input id="name-input" type="text" />
+        <form id="nameForm">
+          <input id="nameInput" type="text" />
```

Comment:
When you have the id based on the HTML, the convention is to use them as kebab-case. If the id was made in JavaScript then we should use only camelCase. (we did the same and then Matilda pointed that out for us).